Cql/Cql.Compiler/ExpressionBuilderContext.cs

```diff
@@ -236,8 +204,8 @@ internal ExpressionBuilderContext WithImpliedAlias(string aliasName, Expression
         /// </summary>
         internal ExpressionBuilderContext Deeper(elm.Element expression)
         {
+            _ = expression;
```

Comment:
It is a no-op, otherwise the compiler complains. Originally the expression was added to a list of predecessors, which in itself wasn't used. I will remove it now, but for the future it would make sense to reintroduce this again so that we can better build exception messages with the expression path that got us to that.

Reply:
I think I'll just remove this from the original code as well

Reply:
Done